inc/customizer/controls/layouts.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class Primer_Customizer_Layouts_Control extends WP_Customize_Control {
+
+	/**
+	 * Enqueue some custom css for this control
+	 */
+	public function enqueue() {
+
+		$rtl    = is_rtl() ? '-rtl' : '';
+		$suffix = SCRIPT_DEBUG ? '' : '.min';
+
+		wp_enqueue_style( 'primer-layouts', get_template_directory_uri() . "/assets/css/admin/layouts{$rtl}{$suffix}.css", array(), PRIMER_VERSION );
+
+	}
+
+	/**
+	 * Override default template with a custom one
```

Comment:
@jonathanbardo This description tells what the user is doing instead of what the method is doing. A more accurate description might be something like: "Display layout choices in the Customizer." Also since you are using a global in this method a good practice would be to add `@global Primer_Customizer_Layouts $primer_customizer_layouts`.